catalogs/catalog-hadoop/src/test/java/org/apache/gravitino/catalog/hadoop/integration/test/HadoopGCSCatalogIT.java

```diff
@@ -95,10 +96,8 @@ protected String defaultBaseLocation() {
 
   protected void createCatalog() {
```

Comment:
Can we move these ITs to each bundle?

Reply:
Yes, I can try it but I will not do it in this PR, I will make this PR large and hard to test the changes introduced by this PR, so I created https://github.com/apache/gravitino/issues/5325 to follow it.